[ID] Update and Reword RC, BN Expectations, Account Restrictions and Ranking Procedure

rorre opened the pull request:
small update to wording and new stuffs from previous PRs

hikikuman reviewed (changes requested):
Tag outdated dari keempat artikel jangan lupa dihapus kalo memang udah nggak outdated lagi.

Lagi itu sempet diobrolin sama Niva di beberapa PR yang lalu-lalu:
- Kriteria Ranking > Ranking Criteria (https://github.com/ppy/osu-wiki/pull/4943#discussion_r573094313)
- Pembatasan/sejenisnya > Restrict (https://github.com/ppy/osu-wiki/pull/4878#discussion_r573912457)
- Dibungkam/sejenisnya > Silence

Mungkin dari tiga poin diatas, bisa diperiksa ulang terus diubah lagi sesuai sama poin2 diatas.

rorre commented:
ah, will glance over the articles again in the following days

AncuL001 commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -27,16 +29,10 @@ Sangat dianjurkan untuk menerima mod sebelum meminta nominasi dari BN, lima hype
-Beatmap yang memiliki 2 nominasi akan dipindahkan ke kategori beatmap [Qualified](/wiki/Beatmap/Category#qualified) selama 7 hari. Qualified Beatmap memiliki scoreboard tapi tidak akan menghasilkan [performance points](/wiki/Performance_Points).
+Beatmap yang memiliki 2 nominasi dari masing-masing game mode yang ada di beatmapset tersebut akan dipindahkan ke kategori beatmap [Qualified](/wiki/Beatmap/Category#qualified) selama 7 hari. Qualified Beatmap memiliki scoreboard tapi tidak akan menghasilkan [performance points](/wiki/Performance_Points). Ketika beatmap berada di Qualified setidaknya selama 7 hari, [antrian ranking](Ranking_queue ...
 
 Kategori ini ada untuk memberikan tanggapan-tanggapan dari Komunitas osu! dengan berharap menemukan potensi yang bermasalah. Masalah tersebut dapat dilaporkan kepada anggota BN dan NAT pada halaman diskusi beatmap. Berbeda dengan Pending beatmaps, Qualified beatmaps tidak dapat diperbarui oleh pembuat, jadi umpan balik atau kritik dapat diterapkan setelah meminta sebuah [pengulangan nominasi](#pe ...
```
Kategori ini ada untuk memberikan tanggapan-tanggapan dari Komunitas osu! dengan harapan potensi masalah dapat ditemukan. Masalah tersebut dapat dilaporkan kepada anggota BN dan NAT pada halaman diskusi beatmap. Berbeda dengan Pending beatmap, Qualified beatmap tidak dapat diperbarui oleh pembuat, jadi umpan balik atau kritik dapat diterapkan setelah meminta sebuah [pengulangan nominasi](#pengulangan-nominasi).

AncuL001 commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -27,16 +29,10 @@ Sangat dianjurkan untuk menerima mod sebelum meminta nominasi dari BN, lima hype
-Beatmap yang memiliki 2 nominasi akan dipindahkan ke kategori beatmap [Qualified](/wiki/Beatmap/Category#qualified) selama 7 hari. Qualified Beatmap memiliki scoreboard tapi tidak akan menghasilkan [performance points](/wiki/Performance_Points).
+Beatmap yang memiliki 2 nominasi dari masing-masing game mode yang ada di beatmapset tersebut akan dipindahkan ke kategori beatmap [Qualified](/wiki/Beatmap/Category#qualified) selama 7 hari. Qualified Beatmap memiliki scoreboard tapi tidak akan menghasilkan [performance points](/wiki/Performance_Points). Ketika beatmap berada di Qualified setidaknya selama 7 hari, [antrian ranking](Ranking_queue ...
 
 Kategori ini ada untuk memberikan tanggapan-tanggapan dari Komunitas osu! dengan berharap menemukan potensi yang bermasalah. Masalah tersebut dapat dilaporkan kepada anggota BN dan NAT pada halaman diskusi beatmap. Berbeda dengan Pending beatmaps, Qualified beatmaps tidak dapat diperbarui oleh pembuat, jadi umpan balik atau kritik dapat diterapkan setelah meminta sebuah [pengulangan nominasi](#pe ...
```
```suggestion
Kategori ini ada untuk memberikan tanggapan-tanggapan dari Komunitas osu! dengan harapan potensi masalah dapat ditemukan. Masalah tersebut dapat dilaporkan kepada anggota BN dan NAT pada halaman diskusi beatmap. Berbeda dengan Pending beatmap, Qualified beatmap tidak dapat diperbarui oleh pembuat, jadi umpan balik atau kritik dapat diterapkan setelah meminta sebuah [pengulangan nominasi](#pengulangan-nominasi).
```

AncuL001 commented on wiki/People/The_Team/Beatmap_Nominators/Expectations/id.md:
```diff
@@ -14,7 +14,7 @@ Sebagai pengingat, Beatmap Nominator diharapkan melakukan yang terbaik dengan se
 - **Jangan mengeksploitasi atau menyalahgunakan sistem yang ada.** Mengikuti [Kriteria Ranking](/wiki/Ranking_Criteria) dan tidak mencoba untuk mengelaknya itu sudah pasti, dan kasus ketidaksepakatan harus diselesaikan melalui proposal Kriteria Ranking. Beatmap Nominator juga tidak boleh berulang kali mengeksploitasi bug atau sistem penghargaan seperti [skor](/wiki/Score), [PP](/wiki/Performance_ ...
 - **Bersikap sopan, baik di dalam maupun di luar platform gim secara langsung.** Beatmap Nominator diharapkan tidak terlalu antagonis, menghasut, atau menghina anggota komunitas osu! lainnya. Hal ini terutama berlaku bagi komunitas dimana pengguna lain biasa mengaitkannya ke akun mereka di osu! (misalnya server mapping di Discord atau Twitter).
-- **Mod secara teratur.** Beatmap Nominator tidak hanya bertanggung jawab untuk memperbaiki unrankable, tetapi juga membuat map yang mereka minati tersebut siap untuk ranked, dan memastikan kualitasnya. Mereka harus selalu waspada dan siap untuk melakukan mod secara menyeluruh, sama seperti yang mereka lakukan saat melamar menjadi BN.
+- **Mod secara teratur.** Beatmap Nominator diharapkan dapat memastikan dan meningkatkan tidak hanya kepastian rank tetapi juga kualitas map yang mereka nominasi. Mereka harus dapat melakukan mod secara menyeluruh jika diperlukan. Mod yang dilakukan di luar diskusi beatmap harus tetap dapat diakses sebagai bukti (misalnya melalui log/ringkasan).
```
```suggestion
- **Mod secara teratur.** Beatmap Nominator diharapkan dapat memastikan dan meningkatkan tidak hanya kepastian rank, tetapi juga kualitas map yang mereka nominasi. Mereka harus dapat melakukan mod secara menyeluruh jika diperlukan. Mod yang dilakukan di luar diskusi beatmap harus tetap dapat diakses sebagai bukti (misalnya melalui log/ringkasan).
```

AncuL001 commented on wiki/Ranking_Criteria/id.md:
```diff
@@ -35,6 +35,7 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
 
 ### Pedoman
 
+- **Pencipta dari segala media yang digunakan dalam beatmapset harus diberi kredit dalam deskripsi beatmapset.** Sesuai dengan [Pedoman Penggunaan Konten](/wiki/Rules/Content_Usage_Guidelines), media yang digunakan dalam beatmap harus diizinkan untuk digunakan di osu!, dan kebijakan lisensinya biasanya merinci mengenai cara memberi kredit kepada penciptanya. Alasan umum untuk menghilangkan kredit ...
```
```suggestion
- **Pencipta dari segala media yang digunakan dalam beatmapset harus diberi pengakuan dalam deskripsi beatmapset.** Sesuai dengan [Pedoman Penggunaan Konten](/wiki/Rules/Content_Usage_Guidelines), media yang digunakan dalam beatmap harus diizinkan untuk digunakan di osu! dan kebijakan lisensinya biasanya merinci mengenai cara memberi pengakuan kepada penciptanya. Alasan umum untuk menghilangkan pengakuan dari deskripsi yaitu sulitnya mencari sumber, keinginan pencipta untuk tidak diberi pengakuan, dan adanya pengakuan di kolom artis, judul, atau sumber.
```
what do you think kredit means lmao

rorre replied:
tbh gw agak bingung itu mau pake apaan so...yeah, changed.

instance lain yang pake itu juga bahasanya dirubah biar lebih mudah dipahami.

rorre commented:
Alright, relooked at the document and have reworded a few things and fixed all the terms mentioned by hikikuman.

AncuL001 commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -1,49 +1,40 @@
 Pengulangan nominasi *(nomination reset)* membantu memastikan bahwa modder, BN dan anggota NAT memeriksa versi terbaru dari beatmapset sebelum memasuki antrian ranking.
 
-Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN atau NAT menemukan sebuah masalah di dalam beatmapset yg ternominasi. Mereka juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) ketika alasannya beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset memenu ...
+Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN atau NAT menemukan sebuah masalah di dalam beatmapset yg ternominasi. Mereka juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) apabila beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset sedang berada d ...
```
```suggestion
Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN atau NAT menemukan sebuah masalah di dalam beatmapset yg ternominasi. Mereka juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) apabila beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset sedang berada di Qualified ketika nominasinya disetel ulang, beatmapset tersebut akan didiskualifikasi dan dihapus dari antrian ranking.
```

AncuL001 commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -1,49 +1,40 @@
+Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN atau NAT menemukan sebuah masalah di dalam beatmapset yg ternominasi. Mereka juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) apabila beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset sedang berada d ...
 
-Anggota dari BN dan NAT kadang melakukan [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) terhadap beatmapset yang mereka yakini memiliki masalah kualitas yang tidak terkait dengan Kriteria Ranking.
+Anggota dari BN dan NAT kadang melakukan [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) terhadap beatmapset yang mereka yakini memiliki masalah kualitas yang tidak terkait dengan Ranking Criteria.
```
```suggestion
Anggota dari BN dan NAT terkadang melakukan [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) terhadap beatmapset yang mereka yakini memiliki masalah kualitas yang tidak terkait dengan Ranking Criteria.
```

AncuL001 commented on wiki/Help_Centre/Account_Restrictions/id.md:
```diff
@@ -1,15 +1,10 @@
-outdated: true
-outdated_since: 2b3040251f1f4876dba46e9c7e01045a7c8ccfc0
----
+# Akun Restriction
```
```suggestion
# Restriction Akun
```
with indonesian language grammar, Restriction Akun is the correct one. i won't fix the rest lol

AncuL001 commented on wiki/Help_Centre/Account_Restrictions/id.md:
```diff
@@ -1,15 +1,10 @@
+Akun restriction adalah tindakan pembatasan dari komunitas yang diterapkan pada akun yang menunjukkan perilaku tidak wajar, mencurigakan, atau melanggar aturan.
 
-Pembatasan akun adalah tindakan pembatasan dari komunitas yang diterapkan pada akun yang menunjukkan perilaku tidak wajar, mencurigakan, atau melanggar aturan.
+Ketika suatu akun ter-restrict, akun tersebut tidak dapat berinteraksi dengan orang lain sampai restrict-nya dicabut. Pemain yang di-restrict masih bisa memainkan game, mengunduh beatmap, dan mengirimkan skor, tetapi profil mereka tidak akan terlihat oleh orang lain.
```
```suggestion
Ketika suatu akun ter-restrict, akun tersebut tidak dapat berinteraksi dengan orang lain sampai restriction-nya dicabut. Pemain yang di-restrict masih bisa memainkan game, mengunduh beatmap, dan mengirimkan skor, tetapi profil mereka tidak akan terlihat oleh orang lain.
```

AncuL001 commented on wiki/Ranking_Criteria/id.md:
```diff
@@ -78,9 +79,9 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
+- **Pembuat [guest difficulty](/wiki/Beatmap/Guest_difficulty), storyboard, skin dan hitsound harus ditambahkan ke tags beatmapset.** Hal ini bertujuan memberikan pengakuan dan membantu orang lain mengidentifikasi kontributor utama dari beatmapset. Nama pengguna yang berisi karakter tunggal dipisahkan oleh spasi harus mengganti spasi dengan garis bawah.
 - **[Sumber metadata utama](/wiki/Beatmap/Primary_metadata_source) harus dijadikan referensi metadata.** Jangan mengubah metadata dari sumber utama kecuali untuk mematuhi aturan pemformatan dan standardisasi di bawah. Jika tidak ada sumber yang tersedia, gunakan sumber yang paling umum dan dapat dikenali.
-- **Artis lagu harus dapat dilacak ke orang yang ada.** Jika tidak ada orang yang tercatat sebagai artis, maka gunakan `Unknown Artist`. Karakter atau program fiksi seperti Vocaloid tidak dapat digunakan sebagai artis tunggal dari lagu.
+- **Artis lagu harus dapat dilacak ke orang yang ada.** Jika tidak ada orang yang tercatat sebagai artis, maka gunakan `Unknown Artist`. Artis tunggal sebuah lagu tidak boleh merupakan suatu karakter atau program fiksi kecuali jika itu adalah alias artis.
```
```suggestion
- **Artis lagu harus dapat dilacak ke orang yang ada.** Jika tidak ada orang yang tercatat sebagai artis, gunakan `Unknown Artist`. Artis tunggal sebuah lagu tidak boleh merupakan suatu karakter atau program fiksi kecuali jika itu adalah alias artis.
```

AncuL001 commented on wiki/Ranking_Criteria/id.md:
```diff
@@ -3,15 +3,15 @@ outdated: true
 outdated_since: f2a2541a7f3463857bb378af0c07814ad7faf405
```
make sure you remove the outdated tag

AncuL001 reviewed (changes requested):
i hope i don't miss anything

rorre commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -1,49 +1,40 @@
+[Modding](/wiki/Modding) adalah proses dari kegiatan yang bertujuan untuk menerima dan membangun kritik guna meningkatkan kualitas sebuah beatmap. Pengguna biasanya mengirimkan umpan balik mereka pada laman diskusi beatmap yang tersedia atau melakukan diskusi secara langsung dengan mapper beatmap yang tersebut.
 
-Para beatmapper baru biasanya membutuhkan modding ekstensif supaya beatmap yang mereka buat memenuhi standar kualitas kategori Ranked dan tidak jarang mereka mendapat kritik seperti lebih baik membuat ulang beatmap mereka dari awal.
+Para beatmapper baru biasanya membutuhkan modding ekstensif supaya beatmap yang mereka buat memenuhi standar kualitas kategori Ranked dan tidak jarang mereka mendapat kritik untuk membuat ulang beatmap mereka dari awal.
```
`seorang` kayaknya agak aneh deh, soalnya ini refer ke grup gitu daripada cuma satu objek, jadinya `Mapper baru biasanya...`

rorre commented:
Applied, a few things though:

- `seorang` kayaknya agak aneh deh, soalnya ini refer ke grup gitu daripada cuma satu objek, jadinya `Mapper baru biasanya...`
- keeping *BNs* as-is cuz english, title mentioned there is also in english so
- added `Oleh` before `karena itu` in suggestion

Also yeah, thought of `Restriction Akun` but everyone else in the channel went with `Akun Restriction` and I'm like `???`, changed to that tho

mousesalt commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -1,49 +1,40 @@
 
+*Lihat juga: [Rank (disambiguasi)](/wiki/Disambiguation/Rank)*
+
 [Beatmap](/wiki/Beatmap) dapat dipromosikan ke Komunitas dengan mengikuti prosedur beatmap ranking dan sudah berada dalam kategori [Ranked](/wiki/Beatmap/Category#ranked).
```
```suggestion
[Beatmap](/wiki/Beatmap) dapat dipromosikan ke Komunitas dengan mengikuti prosedur beatmap ranking dan siap memasuki dalam kategori [Ranked](/wiki/Beatmap/Category#ranked).
```

mousesalt replied:
on the main page, the word `Entering` is a present participle.

mousesalt commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -1,49 +1,40 @@
-Para beatmapper baru biasanya membutuhkan modding ekstensif supaya beatmap yang mereka buat memenuhi standar kualitas kategori Ranked dan tidak jarang mereka mendapat kritik seperti lebih baik membuat ulang beatmap mereka dari awal.
+Mapper baru biasanya membutuhkan modding ekstensif supaya beatmap yang mereka buat memenuhi standar kualitas kategori Ranked. Tidak jarang mereka mendapat kritik untuk membuat ulang beatmap mereka dari awal.
 
 Sebelum sebuah beatmap dapat dinominasikan untuk ranking, beatmapset membutuhkan postingan [hype](/wiki/Glossary#hype) dari lima pengguna berbeda.
```
```suggestion
Sebelum sebuah beatmap dapat dinominasikan untuk ranking, beatmapset membutuhkan postingan [hype](/wiki/Glossary#hype) minimal dari lima pengguna berbeda.
```

mousesalt commented on wiki/Help_Centre/Account_Restrictions/id.md:
```diff
@@ -1,15 +1,10 @@
-Ketika akses suatu akun dibatasi, akun tersebut tidak dapat berinteraksi dengan orang lain sampai sanksinya dicabut. Pemain yang dibatasi masih bisa memainkan game, mengunduh beatmap, dan mengirimkan skor, tetapi profil mereka tidak akan terlihat oleh orang lain.
-
-Fitur berikut dinonaktifkan untuk akun yang dibatasi:
+Fitur berikut dinonaktifkan untuk akun yang ter-restrict:
```
```suggestion
Berikut fitur-fitur yang dinonaktifkan untuk akun yang terkena restrict:
```

mousesalt commented on wiki/Help_Centre/Account_Restrictions/id.md:
```diff
@@ -18,85 +13,85 @@ Fitur berikut dinonaktifkan untuk akun yang dibatasi:
 - memperbarui detail profil
```
```suggestion
- perbaruan detail profil
```

mousesalt commented on wiki/People/The_Team/Beatmap_Nominators/Expectations/id.md:
```diff
@@ -14,7 +9,7 @@ Sebagai pengingat, Beatmap Nominator diharapkan melakukan yang terbaik dengan se
 - **Bertindak dengan mengutamakan kepentingan komunitas.** Dekati situasi yang sulit dengan berhati-hati, tujukan tindakan dengan penilaian yang tepat jika memungkinkan, berusaha untuk belajar dari kesalahanmu, dan sepanjang waktu, pastikan kamu memikirkan dampak dari tindakanmu terhadap komunitas secara luas.
```
```suggestion
- **Bertindak dengan mengutamakan kepentingan komunitas.** Dekati situasi yang sulit dengan hati-hati, tujukan tindakan dengan penilaian yang tepat jika memungkinkan dengan berusaha untuk belajar dari kesalahan sepanjang waktu dan pastikan juga memikirkan dampak dari tindakanmu terhadap komunitas secara luas.
```

mousesalt commented on wiki/Ranking_Criteria/id.md:
```diff
@@ -1,17 +1,12 @@
+# Ranking criteria
 
-# Kriteria ranking
+*Untuk ranking criteria spesifik [mode permainan](/wiki/Game_mode), kunjungi: [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), dan [osu!mania](osu!mania)*
```
```suggestion
*Untuk kriteria ranking [mode permainan](/wiki/Game_mode) yang spesifik, kunjungi: [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), dan [osu!mania](osu!mania)*
```

mousesalt replied:
this refers to the line above
if you found similar case (to be `kriteria ranking`), please fix

mousesalt commented on wiki/Ranking_Criteria/id.md:
```diff
@@ -60,7 +56,7 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
-- **Seorang [beatmapset host](/wiki/Beatmap/Beatmap_host) harus membuat tingkat kesulitan yang jumlahnya sama atau lebih dari pembuat [guest difficulty](/wiki/Beatmap/Guest_difficulty).** hal ini bertujuan untuk memberikan kredit. [Tingkat kesulitan kolaboratif](/wiki/Beatmap/Beatmap_collaborations) hanya dianggap sebagai sebagian dari tingkat kesulitan, dan [drain time](/wiki/Gameplay/Drain_time ...
+- **Seorang [beatmapset host](/wiki/Beatmap/Beatmap_host) harus membuat tingkat kesulitan yang jumlahnya sama atau lebih dari pembuat [guest difficulty](/wiki/Beatmap/Guest_difficulty).** hal ini bertujuan untuk memberi pengakuan. [Tingkat kesulitan kolaboratif](/wiki/Beatmap/Beatmap_collaborations) hanya dianggap sebagai sebagian dari tingkat kesulitan, dan [drain time](/wiki/Gameplay/Drain_time ...
 - **Seorang [beatmapset host](/wiki/Beatmap/Beatmap_host) dan pembuat [guest difficulty](/wiki/Beatmap/Guest_difficulty) dapat mengubah tingkat kesulitan masing-masing sesuai keinginan.** Jika ada ketidaksepakatan antara keduanya, host beatmapset harus menghapus guest difficulty atas permintaan. Jika pembuat guest difficulty tidak dapat dihubungi selama sebulan, mereka akan dianggap setuju dengan ...
 
```
since it been added, add this one.
```- **Beatmapsets tidak boleh berisi bagian-bagian signifikan dari tingkat kesulitan milik pengguna lain tanpa keterlibatan atau izin eksplisit dari mereka.**```

mousesalt commented:
fix this located on beatmap ranking procedure last paragraph
![image](https://user-images.githubusercontent.com/39128490/117989371-753e8600-b366-11eb-982a-5ac890c8bda0.png)
`perfomance` to `performance`

mousesalt commented on wiki/Ranking_Criteria/id.md:
```diff
@@ -95,7 +92,7 @@ Perubahan aturan dan pedoman dalam dokumen ini diusulkan dan dibahas di [forum R
 - **Segala bentuk `vs.`, `Vs.`, `VS`, dll. harus ditulis sebagai `vs.` ketika digunakan sebagai penanda kolaborasi antara dua atau lebih artis.**
 - **Segala bentuk `feat.`, `ft.`, `Ft.`, etc. harus ditulis sebagai `feat.` ketika digunakan sebagai penanda tamu artis dalam lagu tersebut.**
-- **Jika karakter fiksi dikreditkan sebagai penyanyi lagu, kredit mereka akan dirubah dalam format `Karakter (CV: Aktor Suara)`.** Untuk aksi langsung, beri kredit pada pengisi suara saja.
+- **Jika karakter fiksi digunakankan sebagai penyanyi lagu, nama mereka akan dirubah dalam format `Karakter (CV: Aktor Suara)`.** Untuk laga hidup, gunakan pengisi suara saja.
```
```suggestion
- **Jika karakter fiksi digunakan sebagai penyanyi lagu, nama mereka harus dirubah ke dalam format `Nama Karakter (CV: Aktor Suara)`.** Untuk laga hidup, gunakan pengisi suara saja.
```

mousesalt commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -1,52 +1,43 @@
 Pengulangan nominasi *(nomination reset)* membantu memastikan bahwa modder, BN dan anggota NAT memeriksa versi terbaru dari beatmapset sebelum memasuki antrian ranking.
 
-Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN atau NAT menemukan sebuah masalah di dalam beatmapset yg ternominasi. Mereka juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) ketika alasannya beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset memenu ...
+Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN atau NAT menemukan sebuah masalah di dalam beatmapset yg ternominasi. Mereka juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) apabila beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset sedang berada d ...
```
```suggestion
Nominasi dapat disetel ulang ketika pembuat beatmapset melakukan pembaruan atau ketika anggota dari BN maupun NAT menemukan sebuah masalah di dalam beatmapset yang telah dinominasi. Beatmapset juga dapat disetel ulang oleh anggota dari [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) apabila beatmap yang dimaksud mengandung konten yang tidak pantas. Jika sebuah beatmapset sedang berada di Qualified ketika nominasinya disetel ulang, beatmapset tersebut akan didiskualifikasi dan dihapus dari antrian ranking.
```

mousesalt replied:
the repeating `atau` seems ambiguous here
yeah don't think the word has to shorten here

mousesalt commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -1,52 +1,43 @@
-### Pindah ke Ranked
-
-Perlu dipahami bahwa beatmapset yang sudah Qualified telah ditempatkan di dalam antrian untuk ranking. 8 beatmapset dari beberapa jenis permainan tersebut akan dipindahkan dari Qualified menuju Ranked, selama mereka sudah berada di dalam antrian selama 7 hari tanpa adanya laporan masalah. Dengan ini bahkan beberapa beatmapset yang tersisa dari qualified memiliki waktu yang lebih sedikit untuk men ...
+Beatmap yang memiliki 2 nominasi dari masing-masing mode permainan yang ada di beatmapset tersebut akan dipindahkan ke kategori beatmap [Qualified](/wiki/Beatmap/Category#qualified) selama 7 hari. Qualified beatmap memiliki scoreboard tapi tidak akan menghasilkan [performance points](/wiki/Performance_Points). Ketika beatmap berada di Qualified setidaknya selama 7 hari, [antrian ranking](Ranking_ ...
```
```suggestion
Beatmap yang memiliki 2 nominasi dari masing-masing mode permainan yang ada di beatmapset tersebut akan dipindahkan ke kategori beatmap [Qualified](/wiki/Beatmap/Category#qualified) selama 7 hari. Qualified beatmap memiliki papan skor tapi tidak akan menghasilkan [performance points](/wiki/Performance_Points). Ketika beatmap berada di Qualified setidaknya selama 7 hari, [antrian ranking](Ranking_queue) dapat memindahkannya ke [Ranked](#ranked).
```

mousesalt replied:
referring to line 41, while using `papan peringkat` shall be consistent

mousesalt commented on wiki/Help_Centre/Account_Restrictions/id.md:
```diff
@@ -1,102 +1,97 @@
-Fitur berikut dinonaktifkan untuk akun yang dibatasi:
+Berikut fitur-fitur yang dinonaktifkan untuk akun yang ter-restrict:
 
 - Obrolan dalam game
```
```suggestion
- obrolan dalam game
```

mousesalt replied:
have to be lowercase to be consistent with others I think

mousesalt reviewed (commented):
i think rest is fine

TicClick commented:
please adjust your translation to #5530 and 596efd8dff27df4ce19401c51c3f809562ea6e95 (these are different parts of the same change)

DoryanR commented on wiki/Beatmap_ranking_procedure/id.md:
```diff
@@ -1,52 +1,43 @@
+Mapper baru biasanya membutuhkan modding ekstensif supaya beatmap yang mereka buat memenuhi standar kualitas kategori Ranked. Tidak jarang mereka mendapat kritik untuk membuat ulang beatmap mereka dari awal.
 
-Sebelum sebuah beatmap dapat dinominasikan untuk ranking, beatmap membutuhkan postingan [hype](/wiki/Glossary#hype) dari lima pengguna berbeda.
+Sebelum sebuah beatmap dapat dinominasikan untuk ranking, beatmapset membutuhkan postingan [hype](/wiki/Glossary#hype) paling sedikit  dari lima pengguna berbeda.
```
```suggestion
Sebelum sebuah beatmap dapat dinominasikan untuk ranking, beatmapset membutuhkan postingan [hype](/wiki/Beatmap/Hype) paling sedikit  dari lima pengguna berbeda.
```

rorre commented:
@TicClick should I also apply `beatmapset` -> `beatmap` | `beatmap` -> `difficulty` on other pages as well? the english one hasn't yet been changed, though, only RC.

TicClick commented:
why, they have been: https://github.com/ppy/osu-wiki/pull/5495. if you need to apply it to any Indonesian articles, please do

rorre commented:
Turns out I was looking at the wrong tab 🤦‍♂️ All should be good now.

Though heads up--Seems like https://github.com/TicClick/osu-wiki/blob/master/wiki/Beatmap_ranking_procedure/en.md still has the term `beatmapsets` being used on last paragraph, first word, might want to look again if it's intentional or not 🤔

TicClick commented:
being my miss, it isn't. will clean that up
